core/domain/search_services.py

```diff
@@ -228,7 +228,7 @@ def search_collections(query, limit, sort=None, cursor=None):
             function will return a cursor to get the next page.
 
     Returns:
-        A 2-tuple with the following elements:
+        (tuple). A 2-tuple with the following elements:
```

Comment:
I don't think this is how we typically represent tuple typeinfos. Please check other examples in the codebase.